conference.js

```diff
@@ -140,13 +140,22 @@ function createInitialLocalTracksAndConnect(roomName) {
                 mediaPermissionPromptVisibilityChanged(true, browser))
     );
 
+    const noVideo = !!config._noVideoDevice;
```

Comment:
Also if this option is going to be used only for testing maybe we should add "test" or "debug" in the name to be clear what is the purpose of the prop. Some comments would help too. WDYT?